<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObjectWheel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>520</width>
      <height>570</height>
     </rect>
    </property>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>510</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#backButton {
	border-image: url(:/pics/back.png);
 }
QPushButton#backButton:pressed {
	border-image: url(:/pics/back2.png);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardButton">
     <property name="geometry">
      <rect>
       <x>253</x>
       <y>510</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#forwardButton {
	border-image: url(:/pics/forward.png);
 }
QPushButton#forwardButton:pressed {
	border-image: url(:/pics/forward2.png);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="optionsLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>290</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>518</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#saveButton {
	border-image: url(:/pics/save.png);
 }
QPushButton#saveButton:pressed {
	border-image: url(:/pics/save2.png);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="masterBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>300</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Master Records</string>
     </property>
     <widget class="QListWidget" name="masterRecordList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="loadMasterButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Load Record</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteMasterButton">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>230</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete Record</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>201</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Toolbox</string>
     </property>
     <widget class="QToolBox" name="widgetBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>251</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="buttonPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>164</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Buttons</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>164</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Input Widgets</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="containerPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>164</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Containers</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="Line" name="toolboxLine">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>3</width>
       <height>500</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="hortLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>290</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="vertLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>3</width>
       <height>500</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteTempButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>520</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#deleteTempButton {
	border-image: url(:/pics/delete.png);
 }
QPushButton#deleteTempButton:pressed {
	border-image: url(:/pics/delete2.png);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="designTitle">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>10</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Design Area</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
